src/challenges/RandomColorGenerator.tsx

```diff
@@ -1,15 +1,39 @@
-import { Container, Stack, Typography } from "@mui/material";
+import { Button, Container, Stack, Typography, Box } from "@mui/material";
 // Add required imports here
+import { useState } from "react";
 export default function RandomColorGenerator() {
-  // Add your code here
+  
+  const [color, setColor] = useState('#000000');
+
+  function hexa(){
+    let num;
+    const color = [];
```

Comment:
Instead of pushing to an array and then turning it into a string, I would like for the function to be concatenating strings :)